From d65377b492b920767adde0c09af3acbf44e13035 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Fri, 11 Sep 2020 22:37:02 +0200
Subject: [PATCH] include: add additional sizes.

References: bsc#1094244 ltc#168122
Patch-mainline: never - added as part of other changes

These are added in commits f2b9ba871beb9 and 8ff80fbe7e987 as part of
bigger changes.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 include/linux/sizes.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/include/linux/sizes.h b/include/linux/sizes.h
index ce3e8150c174..9874f6f67537 100644
--- a/include/linux/sizes.h
+++ b/include/linux/sizes.h
@@ -1,13 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
 /*
  * include/linux/sizes.h
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
  */
 #ifndef __LINUX_SIZES_H__
 #define __LINUX_SIZES_H__
 
+#include <linux/const.h>
+
 #define SZ_1				0x00000001
 #define SZ_2				0x00000002
 #define SZ_4				0x00000004
@@ -44,4 +43,7 @@
 #define SZ_1G				0x40000000
 #define SZ_2G				0x80000000
 
+#define SZ_4G				_AC(0x100000000, ULL)
+#define SZ_64T				_AC(0x400000000000, ULL)
+
 #endif /* __LINUX_SIZES_H__ */
-- 
2.28.0

